dev-lang/luau/luau-0.645.ebuild

```diff
@@ -22,17 +22,33 @@ fi
 
 LICENSE="MIT"
 SLOT="0"
-
+IUSE="test"
+RESTRICT="!test? ( test )"
 DOCS=( CONTRIBUTING.md README.md SECURITY.md )
 
 src_test() {
 	"${BUILD_DIR}/Luau.UnitTest" || die
 	"${BUILD_DIR}/Luau.Conformance" || die
 }
 
+src_configure() {
+	local mycmakeargs=(
+		-DLUAU_BUILD_TESTS=$(usex test)
+	)
+	cmake_src_configure
```

Comment:
do the test pass inside the sandbox? (`env FEATURES="test usersandbox network-sandbox" emerge -1v luau`)